Libraries/Microsoft.Teams.Plugins/Microsoft.Teams.Plugins.AspNetCore.BotBuilder/Controllers/MessageController.cs

```diff
@@ -53,10 +50,9 @@ public async Task<IResult> PostAsync()
                 return Results.Empty;
             }
 
-            // Fallback logic
-            var authHeader = HttpContext.Request.Headers.Authorization.FirstOrDefault() ?? throw new UnauthorizedAccessException();
-            var token = new JsonWebToken(authHeader.Replace("Bearer ", ""));
-            var res = await _plugin.Do(new()
+            // Fallback logic use the plugin to process the activity
```

Comment:
Q: what does this fallback logic note mean? From what I can see this isn't fallback logic it is the main logical path to use the plugin to process the activity.

Reply:
If the BF adapter doesn't return an invoke response then the control flow will fallback to Teams sdk app.